doc-content/enterprise-only/admin-and-config/proc-configuring-kie-server-persistence.adoc

```diff
@@ -0,0 +1,66 @@
+[id='proc-configuring-kie-server-persistence_{context}']
+
+= Configuring {KIE_SERVER} Persistence
+
+You can configure the {KIE_SERVER} persistence by passing hibernate or JPA parameters as system properties.
+
+To pass the hibernate or JPA parameters, use the following prefixes in your system properties:
+
+* `javax.persistence`
+* `hibernate`
+
+.Procedure
+
+. To configure {KIE_SERVER} persistence, complete any of the following tasks:
+
++
+--
+If you want to configure {KIE_SERVER} persistence using {EAP} configuration file, complete the following tasks:
+
+... In your {PRODUCT} installation directory, navigate to the `standalone-full.xml` file. For example, if you use {EAP} installation for {PRODUCT}, go to `$EAP_HOME/standalone/configuration/standalone-full.xml` file.
+... Open the `standalone-full.xml` file and under the `<system-properties>` tag, set your hibernate or JPA parameters as system properties.
++
+The following example shows how to configure {KIE_SERVER} persistence using hibernate parameters:
```

Comment:
I'd suggest adding this sentence as a tag to the source as follows:

.Example of configuring {KIE_SERVER} persistence using hibernate parameters
[source]
...

Reply:
Updated